client/components/Nav.tsx

```diff
@@ -1,16 +1,167 @@
+                        edge="start"
+                        color="inherit"
+                        aria-label="menu"
+                        sx={{ mr: 2 }}
+                        aria-controls="menu-appbar"
+                        aria-haspopup="true"
+                        onClick={e => setMenuIconAnchorEl(e.currentTarget)}
+                    >
+                        <MenuIcon />                
+                    </IconButton>
+                    <Menu
+                        anchorEl={menuIconAnchorEl}
+                        anchorOrigin={{
+                            vertical: 'top',
+                            horizontal: 'right',
+                        }}
+                        keepMounted
+                        transformOrigin={{
+                            vertical: 'top',
+                            horizontal: 'right',
+                        }}
+                        open={Boolean(menuIconAnchorEl)}
+                        onClose={e => setMenuIconAnchorEl(null)}
+                    >
+                        <MenuItem 
+                            onClick={e => {
+                                setMenuIconAnchorEl(null);
+                                navigate("/");
+                            }}
+                        > 
+                            Home
+                        </MenuItem>
+                        {!isLoggedIn && (
+                            <>
+                                <MenuItem 
+                                    onClick={e => {
+                                        setMenuIconAnchorEl(null);
+                                        navigate("/login");
+                                    }}
+                                > 
+                                    Login
+                                </MenuItem>
+                                <MenuItem 
+                                    onClick={e => {
+                                        setMenuIconAnchorEl(null);
+                                        navigate("/register");
+                                    }}
+                                > 
+                                    Register
+                                </MenuItem>
+                            </> 
+                        )}
+                    </Menu>
+                    <Box
+                        component="div"
+                        sx={{
+                            display: "flex",
+                            justifyContent: "center"
+                        }}
+                    >
+                        <Typography 
+                            variant="h6" 
+                            component="div"
+                            sx={{pr: 1}} 
+                        >
+                            ZOOP
+                        </Typography>
+                        <Typography
+                            variant="h6"
+                            component="div"
+                        >
```

Comment:
This is a purely presentational element, not a real header. You might want to add an aria label to indicate that it's just for presentation (I forget which label that is), otherwise it may be read out loud as "dash dash dash" in screen readers

Reply:
Hmm. There is no aria label prop for the Typography component. I've been wondering, Should the Zoop header with the paper airplane be a png logo instead?

Reply:
I could wrap the Typography component in a Box that has "div" for its component prop and give that and ariaLabel of "presentation" or "none" (those are the aria-label attribute values for this situation, according to MDN).

Reply:
Never mind. When I set the `Typography` component prop to `div` it has an aria-label prop.

Reply:
> I've been wondering, Should the Zoop header with the paper airplane be a png logo instead?

@paloobi Ashley drew a couple of quick svg images for Zoop. I'll share them in the main comments.